src/util/config/config.cpp

```diff
@@ -381,11 +381,6 @@ namespace dxvk {
       { "d3d9.customVendorId",              "10de" },
       { "d3d9.customDeviceId",              "0402" },
     }} },
-    /* Warhammer: Online                         */
-    { R"(\\WAR(-64)?\.exe$)", {{
-      { "d3d9.allowImplicitDiscard",        "False" },
-      { "d3d9.customVendorId",              "1002" },
```

Comment:
Still needs the vendor ID for avoiding the weird srgb format, doesn't it?

Reply:
Yes. I was a little too eager there.